tests/cephadm/test_cephadm_log_spam.py

```diff
@@ -10,20 +15,52 @@ def run(ceph_cluster, **kw):
               to be used in the test.
     """
     node = ceph_cluster.get_nodes(role="_admin")[0]
+    installer = ceph_cluster.get_nodes(role="installer")[0]
     admin = node.ssh
+
+    # Validate log spam under cephadm.log file
     with admin().open_sftp().open(CEPHADM_LOG_PATH, "r") as file:
         content = file.read()
         # Spam logs were being generated as part of 'gather-facts'
         # Validate 'gather-facts' related logs are present in the log file
         if b"gather-facts" not in content:
-            OperationFailedError(
+            raise OperationFailedError(
                 f"Failed: Expected log 'gather-facts' not found in {CEPHADM_LOG_PATH}"
             )
         # Spam logs had the string 'DEBUG sestatus' in them
         # Validate spam logs are not present in the log file
         if b"DEBUG sestatus:" in content:
-            OperationFailedError(
+            raise OperationFailedError(
                 f"Failed: Spam log 'DEBUG sestatus' found in {CEPHADM_LOG_PATH}"
             )
 
+    # Validate log spam under "cephadm logs" for mgr daemon
+    spam = "Detected new or changed devices"
+    fsid = CephAdm(node).ceph.fsid()
```

Comment:
`fsid` is not used anywhere, why is this check required here?

Reply:
It is used below for the cephadm logs command